dashboard/src/modules/components/ComparisonComponent/index.jsx

```diff
@@ -15,30 +15,29 @@ import React, { useEffect } from "react";
 import {
   compareMultipleDatasets,
   getQuisbyData,
+  resetList,
   toggleCompareSwitch,
 } from "actions/comparisonActions";
 import { useDispatch, useSelector } from "react-redux";
 
 import Cookies from "js-cookie";
 import PanelConent from "./PanelContent";
-import { VISUALIZATION } from "assets/constants/compareConstants";
-import { getDatasets } from "actions/overviewActions";
 
 const ComparisonComponent = () => {
   const dispatch = useDispatch();
   const loggedIn = Cookies.get("isLoggedIn");
 
-  const { datasets } = useSelector((state) => state.overview);
+  const { publicData } = useSelector((state) => state.overview);
   const { isCompareSwitchChecked, selectedResourceIds } = useSelector(
     (state) => state.comparison
   );
   useEffect(() => {
-    if (datasets && datasets.length > 0) {
-      dispatch(getQuisbyData(datasets[0]));
+    if (publicData && publicData.length > 0) {
```

Comment:
Minor point, but the old name seems better since both Results and Comparison now support selection of "my/public/all" ... our datasets may not be "public data".

Reply:
Got it